s3/pkg/datastore/yig/conf/common.toml

```diff
@@ -0,0 +1,7 @@
+[log]
```

Comment:
can we stick with the a=b properties file style for configuration files here, any specific reason to use toml ? My question comes from the user side of things...in OSDS, if some configuration files are property style, some are toml...it is inconsistent. Adding on, imho, updating a placeholder variable is easier in a properties style file using SED etc.